internal/config/get.go

```diff
@@ -0,0 +1,178 @@
+
+func GetIndexerHealthcheckPort() int { return get("INDEXER_HEALTHCHECK_PORT", "", false, toInt) }
+
+func GetPostgresConnectionPoolSize() int {
+	return get("POSTGRES_CONNECTION_POOL_SIZE", "3", true, toInt)
+}
+
+func GetInspectServerQueueSize() int { return get("INSPECT_SERVER_QUEUE_SIZE", "", false, toInt) }
+
+func GetSnapshotDir() string { return get("SNAPSHOT_DIR", "", false, toString) }
+
+func GetGraphqlPort() int { return get("GRAPHQL_PORT", "4000", true, toInt) }
+
+func GetAuthorityAddress() string { return get("AUTHORITY_ADDRESS", "", false, toString) }
+
+func getTxSigningAwsKmsRegion() *string {
+	return getOptional("TX_SIGNING_AWS_KMS_REGION", "", false, toString)
+}
+
+func GetTransactionManagerDatabasePath() string {
+	return get("TRANSACTION_MANAGER_DATABASE_PATH", "", false, toString)
+}
+
+func GetBrokerEndpoint() string { return get("BROKER_ENDPOINT", "redis://redis:6379", true, toString) }
+
+func GetLogEnableColor() bool { return get("LOG_ENABLE_COLOR", "true", true, toBool) }
+
+func GetInputBoxAddress() string { return get("INPUT_BOX_ADDRESS", "TODO", true, toString) }
+
+func getInspectServerPort() *int { return getOptional("INSPECT_SERVER_PORT", "5005", true, toInt) }
+
+func GetStateServerEndpoint() string { return get("STATE_SERVER_ENDPOINT", "", false, toString) }
+
+func GetBlockchainProviderEndpoint() string {
+	return get("BLOCKCHAIN_PROVIDER_ENDPOINT", "", false, toString)
+}
+
+func GetDappBlockHash() string { return get("DAPP_BLOCK_HASH", "", false, toString) }
+
+func getTxSigningMnemonicFile() *string {
+	return getOptional("TX_SIGNING_MNEMONIC_FILE", "", false, toString)
+}
+
+func GetSnapshotEnabled() bool { return get("SNAPSHOT_ENABLED", "true", true, toBool) }
+
+func GetStateServerMaxDecodingMessageSize() string {
+	return get("STATE_SERVER_MAX_DECODING_MESSAGE_SIZE", "", false, toString)
+}
+
+func GetStateFoldConcurrentEventsFetch() string {
+	return get("STATE_FOLD_CONCURRENT_EVENTS_FETCH", "", false, toString)
+}
+
+func GetLogLevel() LogLevel { return get("LOG_LEVEL", "warning", true, toLogLevel) }
+
+func GetInspectServerConfigPath() string {
+	return get("INSPECT_SERVER_CONFIG_PATH", "", false, toString)
+}
+
+func GetBrokerBackoffMaxElapsedDuration() string {
+	return get("BROKER_BACKOFF_MAX_ELAPSED_DURATION", "", false, toString)
+}
+
+func GetStateFoldSafetyMargin() int { return get("STATE_FOLD_SAFETY_MARGIN", "20", true, toInt) }
+
+func GetServerManagerEndpoint() string { return get("SERVER_MANAGER_ENDPOINT", "", false, toString) }
+
+func getTxSigningMnemonic() *string { return getOptional("TX_SIGNING_MNEMONIC", "", false, toString) }
+
+func GetBlockchainDefaultConfirmations() int {
+	return get("BLOCKCHAIN_DEFAULT_CONFIRMATIONS", "10", true, toInt)
+}
+
+func GetDappAddress() string { return get("DAPP_ADDRESS", "", false, toString) }
+
+func GetAuthorityClaimerPort() int { return get("AUTHORITY_CLAIMER_PORT", "", false, toInt) }
+
+// Calls all the <Get> functions.
+func getAll() {
```

Comment:
This function won't work as you expect. Depending on the execution mode, the set of required variables will change. And I don't think the config module should be responsible for that.